cmd/createcluster.go

```diff
@@ -548,3 +567,32 @@ func nextPortFunc(startPort int) func() int {
 		return port
 	}
 }
+
+// validAddr returns a valid checksummed ethereum address. Returns an error if a valid address cannot be constructed.
+func validAddr(a string) (string, error) {
+	if !common.IsHexAddress(a) {
+		return "", errors.New("invalid address")
+	}
+
+	hexAddr := common.HexToAddress(a)
+
+	return hexAddr.Hex(), nil
+}
+
+// validNetwork returns an error if the input network is not supported or certain conditions are not met.
+func validNetwork(conf clusterConfig) error {
+	validNetworks := []string{"prater", "kintsugi", "kiln", "gnosis", "mainnet"}
+
+	// We cannot allow a zero withdrawal address on mainnet or gnosis.
+	if conf.WithdrawalAddr == defaultWithdrawalAddr && (conf.Network == "mainnet" || conf.Network == "gnosis") {
+		return errors.New("zero address", z.Str("network", conf.Network))
```

Comment:
@xenowits error message is not clear, perhaps "forbidden use of zero address in this network" or something like that

Reply:
okay